src/app/home-page/recent-events/recent-events.component.ts

```diff
@@ -86,7 +86,7 @@ export class RecentEventsComponent extends Base implements OnInit {
     private usersService: UsersService,
     private alertService: AlertService,
     private eventsService: EventsService,
-    private orgLogoService: OrgLogoService,
+    public orgLogoService: OrgLogoService,
```

Comment:
Template was accessing a private property, which the compiler flagged (worked at runtime because JavaScript doesn't have private)